### ISSUE TITLE
Add VoiceOver Compatibility

### ISSUE THREAD

**Issue by austinseraphin** [goodtohear/habits#8]
The app does not work at all with VoiceOver. The opening screen displays a heading with the name of the app and a heading with the current date, but nothing else. Use standard buttons, and make sure they all have accessibility labels.
